### DISCUSSION TITLE
How to integrate Java language server with react monaco editor

### DISCUSSION THREAD

**Discussion by chaitanya71998** [TypeFox/monaco-languageclient#442] [Q&A]
Greetings,

I was trying to create a java language multiple editor. To add intellisense and other features I came to know it can be done with [eclipse.jdt.ls](https://github.com/eclipse/eclipse.jdt.ls). But i am not getting idea on how to integrate, create a server with the eclipse JDT language server. What are the steps todo achieve this. i referred to the issues but not able to start the eclipse JDT language server. 
i got like this after following the steps in this [eclipse.jdt.ls](https://github.com/eclipse/eclipse.jdt.ls), but not clear on what port is this running and how to use it.
![image](https://user-images.githubusercontent.com/46353549/209957927-a2e9d53b-6397-46fd-9b60-78069e19d993.png)

Thank you

**Comment by dhruvk1989** [comment]
Did you find anything? I'm struggling too.

**Reply by kaisalmen** [reply]
@chaitanya71998 we have full example available that makes use of `monaco-editor-wrapper` and not the react component, but apart from it is al there: https://github.com/TypeFox/monaco-languageclient#java-language-client-and-language-server-example-location
We circumvent the usual jdt.ls setup problems by using a container.
